### ISSUE TITLE
Improving terms panels - line chart, bottom N, multi-series

### ISSUE THREAD

**Issue by dougmcclure** [lucidworks/banana#136]
Improving the terms panel would be nice. Adding a line chart, bottom N and the ability to plot min/max/avg values here would be very helpful.

**Comment by aadel** [comment]
I think having bottom N is a very nice, but I can't see how line chart fits here. The values returned are discreet (Top N) and they need not be connected anyway, hence pie (for percentage) and bar make sense. Terms panel settings already supports aggregate functions for a specific field.

**Comment by dougmcclure** [comment]
Suggesting a line chart option here not for use with TopN/BottomN but to use with stats. Need a way to draw a line chart with avg/max/mean, etc over time..  Not sure if it's easier to add here or add the stats feature to hisogram panel.

**Comment by aadel** [comment]
Yes, I believe it makes more sense to add aggregate functions to histogram and line charts are already there. This seems related to https://github.com/LucidWorks/banana/issues/135

**Comment by dougmcclure** [comment]
Agree - lets get bottom N added for terms panel and leave the rest of the stats for histogram or new panel types using d3. Kibana 4 has added a lot more stats types - hope we can get more stats/analytics component things added soon.
